### ISSUE TITLE
[BUG]: The HTML report does not include ignored cases as hidden when you put ignored cases in the baselines folder

### ISSUE THREAD

**Issue by kbpitula** [IBMa/equal-access#2319]
### Project

accessibility-checker for Node

### Browser

Chrome

### Operating system

Other

### Description

The HTML report does not include ignored cases as hidden when you put ignored cases in the baselines folder

I added json report to baselinies ( generated previously by a11y checker ) 

When i re-run scan my json report - marked issues from baselinies as "ignored" but HTML not

### Steps to reproduce

How to reproduce:
1. Scan page ( generate json, html report ) 
2. Add generated report json to baselinies 
3. put ignored - from false to true (not so sure is mandatory step ) 
4. re-scan - generate json and html report

My new json report - marked issues as "ignored" -> expected
but HTML not - If i good caught should be marked as "HIDDEN"

**Comment by MHoov** [comment]
From [6/26/25 Slack thread](https://ibm-studios.slack.com/archives/C036P1CTN/p1750947200639229):

>This is something we would like to have in the near future.  Our accessibility policy is more strict and relies on the HTML output. Marking issues as hidden manually each time consumes a lot of time — especially when we have automation in place.  That’s why this feature is very important for us.

**Comment by tombrunet** [comment]
Triage: I'm suggesting high priority if we can get this done in a half a sprint or less. If this looks like it will be larger than that, we'll have to circle back to it.
